### DISCUSSION TITLE
[@types/node] v8.writeHeapSnapshot() takes optional second parameter since node 20

### DISCUSSION THREAD

**Discussion by novedevo** [DefinitelyTyped/DefinitelyTyped#68518] [Issues with a @types package] [answered]
The current type definitions for node 20 don't capture this. See below links for reference.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c39375ac13ffe3a3e4ff65328813c9c56d614d4c/types/node/v8.d.ts#L249
https://nodejs.org/docs/latest-v20.x/api/v8.html#v8writeheapsnapshotfilenameoptions

Thank you for this project!

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Added in PR #68264
